tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "src",
     "target": "es5",
```

Comment:
I don't like to have to do `import MyComponent from '../../my/ui/MyComponent';` if I could afford it...